### PR TITLE
ISPN-6468 Ignore SyncReplExtendedStatisticTest

### PR DESCRIPTION
**Pull request by slaskawi**
To be fixed by https://issues.jboss.org/browse/ISPN-6468

### CODE REVIEW

**Comment by anistor** [comment]
Pulling ..

**Comment by anistor** [comment]
Integrated. Thanks @slaskawi !

File: extended-statistics/src/test/java/org/infinispan/stats/simple/SyncReplExtendedStatisticTest.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * @author Pedro Ruivo
  * @since 6.0
  */
-@Test(groups = "functional", testName = "stats.simple.SyncReplExtendedStatisticTest")
+@Test(groups = "functional", testName = "stats.simple.SyncReplExtendedStatisticTest", enabled = false, description = "To be fixed by ISPN-6468")
```
Comment (danberindei) [inline extended-statistics/src/test/java/org/infinispan/stats/simple/SyncReplExtendedStatisticTest.java]: @slaskawi Did we give up on the `unstable` test group?

**Comment by danberindei** [comment]
@slaskawi @anistor Disabling the test didn't work, probably because the test methods are all defined in a base class. They're still running, but at least now they're failing constantly ;)

http://ci.infinispan.org/project.html?projectId=Infinispan&testNameId=185360471509903651&tab=testDetails

**Comment by anistor** [comment]
I'll move it to unstable then.

**Comment by anistor** [comment]
Moving to unstable has the same dumb effect. Inherited tests still run during normal build, which reminds me why I dislike testng.

**Comment by pruivo** [comment]
@anistor just override the test in ```SyncReplExtendedStatisticTest``` and tag it as unstable

**Comment by anistor** [comment]
On my machine all __8__ of them are failing :) Overriding 8 inherited methods is quite a bit of work :)

**Comment by pruivo** [comment]
only one test was failing randomly, now there are 8????

**Comment by anistor** [comment]
yup, see org.infinispan.stats.simple in this run:  http://ci.infinispan.org/viewLog.html?buildId=38370&tab=buildResultsDiv&buildTypeId=bt9

**Comment by pruivo** [comment]
here you go: https://github.com/infinispan/infinispan/pull/4218

**Pull request closed without merging**